pinot-query-planner/src/main/java/org/apache/pinot/query/type/TypeSystem.java

```diff
@@ -32,6 +32,9 @@ public class TypeSystem extends RelDataTypeSystemImpl {
   private static final int MAX_DECIMAL_SCALE_DIGIT = 1000;
   private static final int MAX_DECIMAL_PRECISION_DIGIT = 1000;
 
+  private static final int DERIVED_DECIMAL_PRECISION = 19;
+  private static final int DERIVED_DECIMAL_SCALE = 1;
```

Comment:
add some comments regarding these 2 constant? 
(also nit can you remove the `_DIGIT` in the previous 2 constants?)